torch/nn/parallel/data_parallel.py

```diff
@@ -1,10 +1,41 @@
 import torch
+import warnings
 from ..modules import Module
 from .scatter_gather import scatter_kwargs, gather
 from .replicate import replicate
 from .parallel_apply import parallel_apply
 
 
+def _check_balance(device_ids):
+    imbalance_warn = """
+    There is an imbalance between your GPUs. You may want to exclude GPU {} which
+    has >25% less memory or cores than GPU {}. You can do so by setting the
```

Comment:
">25% less" is a bit hard to understand. How about "less than 75% of memory or cores ..."